optaplanner-distribution/src/main/assembly/filtered-resources/UpgradeFromPreviousVersionRecipe.txt

```diff
@@ -2791,3 +2791,22 @@ After in *.java:
 
 [README] If you're using JPA-Hibernate with OptaPlanner, you'll want to take advantage of the new module optaplanner-persistence-jpa.
 See docs chapter "Integration" for more information.
+
+[MINOR] If you're using SingleBenchmarkResult: the functionality of the method getScore() and getUninitializedVariableCount() has moved to
+getAverageScore() and getAverageUninitializedVariableCount() because of the addition of SubSingleBenchmarkResult (Statistical benchmarking).
+Before in *.java:
+    singleBenchmarkResult.getScore();
+    singleBenchmarkResult.getUninitializedVariableCount();
+After in *.java:
+    singleBenchmarkResult.getAverageScore();
+    singleBenchmarkResult.getAverageUninitializedVariableCount();
+
+[MINOR] If you're using SingleStatistic or any of it's subclasses: all of them were renamed to include "Sub" as a part of
```

Comment:
I'd change this to an [IMPL_DETAIL]. Avoid any confusion that the <singleStatisticType> elements in benchmark configs need to be renamed too, because they don't. (see earlier comment)